aiosmtpd/controller.py

```diff
@@ -5,12 +5,16 @@
 from aiosmtpd.smtp import SMTP
 from public import public
 from typing import Any, Dict
+# Uncomment next line when we hit 1.3
+# from warnings import warn
```

Comment:
Part of the planned deprecation of the `enable_SMTPUTF8` flag.

The flag is not used by Controller; it's passed-through to the SMTP instance.